src/mpl/include/uthash.h

```diff
@@ -142,8 +142,11 @@ do {
 #define HASH_FIND(hh,head,keyptr,keylen,out)                                     \
 do {                                                                             \
   unsigned _hf_hashv;                                                            \
-  HASH_VALUE(keyptr, keylen, _hf_hashv);                                         \
-  HASH_FIND_BYHASHVALUE(hh, head, keyptr, keylen, _hf_hashv, out);               \
+  (out) = NULL;                                                                  \
```

Comment:
Can you add a note to the comment block at the top explaining this is a change we made? It would probably be good to submit this upstream and see if there's any feedback from the maintainers. Otherwise it looks great!

Reply:
Sure, just added.

Reply:
I should have been more clear. There is a comment block at the top of the file that should contain info on all the MPICH-specific changes. Let's put the comment in there.

Reply:
@raffenet Oh I see. My mistake. I have just updated the comment. Does it look ok?

Reply:
Yes, looks good now.